domain-cc/cc-app/src/python_src/api.py

```diff
@@ -82,6 +82,34 @@ def log_as_json(log: dict):
     logging.info(json.dumps(log))
 
 
+def log_claim_stats(claim: Claim, classification: Optional[PredictedClassification]):
+    if classification:
+        classification_code = classification.classification_code
+        classification_name = classification.classification_name
+    else:
+        classification_code = None
+        classification_name = None
+
+    contention_text = claim.contention_text or ""
+    is_in_dropdown = contention_text.strip().lower() in dropdown_values
+    is_mapped_text = dropdown_lookup_table.get(claim.contention_text, None) is not None
+    sanitized_contention_text = contention_text if is_mapped_text else "unmapped contention text"
+
+    log_as_json(
+        {
+            "claim_id": sanitize_log(claim.claim_id),
+            "claim_type": sanitize_log(claim.claim_type),
+            "classification_code": classification_code,
+            "classification_name": classification_name,
+            "contention_text": sanitized_contention_text,
```

Comment:
Is `contention_text` the same as `lookup_table_match` in `log_lookup_table_match`? Maybe keeping this name consistent would make for less work on the dashboard side?